src/utils/parseEnvironmentVariables.js

```diff
@@ -0,0 +1,18 @@
+import { ENV_VARIABLE_TYPE_SECRET, ENV_VARIABLE_TYPE_VALUE } from '../constants'
+
+export const parseEnvVariables = envVariables => {
+  return envVariables.map(envVariable => {
+    let type = envVariable.value
+      ? ENV_VARIABLE_TYPE_VALUE
+      : ENV_VARIABLE_TYPE_SECRET
+    let value =
+      envVariable.value ??
+      `${envVariable.value_from.secret_key_ref.name}:${envVariable.value_from.secret_key_ref.key}`
+
+    if (envVariable?.value?.includes(':')) {
+      type = ENV_VARIABLE_TYPE_SECRET
+    }
```

Comment:
This is wrong.
The `<secret-name>:<secret-key>` is just an arbitrary way I made up for how to show both values in the single “Value” column.

The way to distinguish between the two is by their structure:
For example, if the array of environment variables looks like this:
```
[
  {
    "name": "my-regular-name",
    "value": "my-regular-value"
  },
  {
    "name": "my-secret-name",
    "valueFrom": {
      "secretKeyRef": {
        "key": "my-secret-key",
        "name": "my-secret-name"
      }
    }
  }
]
```
If it has `value` then it is of type “Value”.
If it has `valueFrom.secretKeyRef` then it is of type “Secret”.

If you try to create a new job, and select “erann-test” function, scroll to the environment variables section and you'll see: 
![image](https://user-images.githubusercontent.com/13918850/137024541-2b3f058a-9484-47e5-8040-44fa9839ed7b.png)
Note that the ones with value of a URL, such as `http://v3io-webapi:8081` are parsed as type “Secret”, when they should be of type “Value”.
If you edit one of them it looks like this:
![image](https://user-images.githubusercontent.com/13918850/137024680-46d35807-e61f-4e46-92bc-1c418390b429.png)